packages/default-theme/components/SwLogin.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div class="sw-login">
+    <div class="form sw-login__form">
+      <h2 class="sw-login__header">Log in</h2>
+      <SfAlert class="sw-login__alert" v-if="error" type="danger" :message="error" />
+      <SfInput
+        v-model="$v.login.$model"
+        name="login"
+        label="Your login"
+        class="form__input"
+        :valid="!$v.login.$error"
+        :disabled="isLoading"
+        errorMessage="Login is required"
+      />
+      <SfInput
+        v-model="$v.password.$model"
+        name="password"
+        label="Password"
+        type="password"
+        class="form__input"
+        :valid="!$v.password.$error"
+        :disabled="isLoading"
+        errorMessage="Password is required"
+      />
+      <SfButton class="sf-button--full-width form__button"
+                @click="invokeLogin"
+                :disabled="isLoading">
+        Login
+      </SfButton>
+    </div>
+  </div>
+</template>
+
+<script>
+import { SfInput, SfButton, SfAlert } from '@storefront-ui/vue'
+import { validationMixin } from 'vuelidate'
+import { required } from 'vuelidate/lib/validators'
+import { useUser } from '@shopware-pwa/composables'
+
+export default {
+  name: 'SwResetPassword',
+  components: { SfButton, SfInput, SfAlert },
+  mixins: [ validationMixin ],
+  data() {
+    return {
+      login: '',
+      password: '',
+    }
+  },
+  setup() {
+    const {login, loading, error} = useUser()
+    return {
+      clientLogin: login,
+      isLoading: loading,
+      error
+    }
+  },
+  validations: {
+    login: {
+      required,
+    },
+    password: {
+      required,
+    }
+  },
+  methods: {
+    async invokeLogin() {
+      this.$v.$touch()
+      if(this.$v.$anyError) {
```

Comment:
```suggestion
      if(this.$v.$invalid {
```
look at this example for the difference https://vuelidate.js.org/#sub-error-vs-any-error